merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/ThreadedTask.java

```diff
@@ -8,21 +8,21 @@
 
 import java.util.Objects;
 import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executor;
 import java.util.function.Supplier;
 
 public final class ThreadedTask<Return> implements Task<Return> {
   private final Scoped<Context> rootContext;
   private final Supplier<Return> task;
-  private final ExecutorService executor;
+  private final Executor executor;
```

Comment:
Wow, I feel stupid. It's nice to know this interface exists 🤦‍♂️

Reply:
I stumbled on it by accident 😅